ros/src/models/chrono/ros_chrono/src/path_following.cpp

```diff
@@ -96,8 +96,8 @@ std::string path_file("paths/switch_lane.txt");
 
 // Rigid terrain dimensions
 double terrainHeight = 0;
-double terrainLength = 300.0;  // size in X direction
-double terrainWidth = 300.0;   // size in Y direction
+double terrainLength = 700.0;  // size in X direction
```

Comment:
move these parameters to YAML file

Reply:
Okay

Reply:
This has been fixed by the commit https://github.com/ChangYuHsuan/MAVs/pull/11/commits/d99b4e373c142b98b9fc4e4eb439f7c423a77285